### ISSUE TITLE
Email domain based registration policy

### ISSUE THREAD

**Issue by LaPingvino** [friendica/friendica#895]
A way to implement this would be to add an independent access level to the domain field.

**Comment by Beanow** [comment]
For example:
gmail.com = open policy
* = approval policy

**Comment by silke** [comment]
We already have this "Allowed email domains" field. (Comma separated list of domains which are allowed in email addresses for registrations to this site. Wildcards are accepted. Empty to allow any domains.)

So this would be an extension to that concept, right?

**Comment by tobiasd** [comment]
Yes, I think so.

**Comment by LaPingvino** [comment]
yes, exactly.

**Comment by realkinetix** [comment]
Is this the same concept as adding in email domains to blacklist for registration?

**Comment by MrPetovan** [comment]
Yes, instead of “allowed_domains” it would be “banned_domains” or something along this.
